FlowCrypt/Functionality/Mail Provider/Message Gateway/GmailService+draft.swift

```diff
@@ -5,70 +5,98 @@
 //  Created by Evgenii Kyivskyi on 10/22/21
 //  Copyright © 2017-present FlowCrypt a. s. All rights reserved.
 //
-import Foundation
+
 import GoogleAPIClientForREST_Gmail
 
 extension GmailService: DraftGateway {
```

Comment:
`Gateway` is something that stuff passes through. Like an `EmailGateway` - you send an email, it passes through, ends up in another email server.

I guess we could call all of these classes just `...ApiClient` ? `DraftApiClient`, `MessagesApiClient`, `SendMessageApiClient` ? I think that would be the clearest. Can leave it for another PR.

In particular I don't want to call everything `...Service` unless it's some sort of background service that keeps running, like a daemon, which we don't have many off.

Reply:
https://github.com/FlowCrypt/flowcrypt-ios/issues/1855